documentation/en/draft_modulators.md

```diff
@@ -0,0 +1,370 @@
+        - [2.5.2.2.2 Oscilador controlado por tensión (VCO).](#25222-oscilador-controlado-por-tensi%c3%b3n-vco)
+      - [2.5.2.3 Modulación por desplazamiento de frecuencia Gausiana (GFSK)](#2523-modulaci%c3%b3n-por-desplazamiento-de-frecuencia-gausiana-gfsk)
+      - [2.5.2.4 Modulación por desplazamiento de fase (**PSK, Phase Shift Keying**)](#2524-modulaci%c3%b3n-por-desplazamiento-de-fase-psk-phase-shift-keying)
+- [Capitulo 3 Filtros](#capitulo-3-filtros)
+  - [3.1 ¿Qué es un filtro de frecuencia?](#31-%c2%bfqu%c3%a9-es-un-filtro-de-frecuencia)
+  - [3.2 Clasificación de los filtros](#32-clasificaci%c3%b3n-de-los-filtros)
+    - [3.2.1 Respuesta en frecuencia](#321-respuesta-en-frecuencia)
+- [Capitulo 4 Demodulador FSK](#capitulo-4-demodulador-fsk)
+
+
+
+<h1>Modulators and signals theory</h1>
+
+Within this chapter we intend to demostrate the principles and fundamental concepts that make possible the wireless interconnection of the nodes, irradiating signals and receiving it, by applying techniques described in this section.
+
+# Chapter 1: Waves
+
+A wave consist in the propationg of disturbance of some property in the space, like:
+
+- Density.
+- Pressure.
+- Electric field.
+- Magnetic field.
+
+Implying an energy transport without matter transport. The disturbed space can contain matter (air, water, etc) or not.
+
+
+<img src="imple_pic/wave.svg" alt="drawing" height="300" width="400" align="block"/>
+
+The example in the figure generating a wave with a cord, it can be appreciate the deformity of the medium in which its represented the energy transport without having to transport the matter itself.
+
+## 1. Characteristic of a wave
+
+
+<figure>
+<img src="imple_pic/wave-characteristics.svg" alt="drawing" height="330" width="400" align="left"/>
+
+<img src="imple_pic/frequency.svg" alt="drawing" height="330" width="400" align="left"/>
+
+</figure>
+
+Una onda puede ser completamente descrita por 5 características llamadas amplitud, frecuencia, longitud de onda y velocidad.
+
+A wave can be described in five characterictics: amplitude, frequency, time, wavelenght, and speed.
+
+
+### 1.1.1 Amplitude (A)
+
+Vertical distance between a crest and the medium point of the wave.
+### 1.1.2 Frequency (F)
+
+The number of vibrations (full cicle) produced in a second is called wave frequency (F). The frequency unit in the International System (IS) its s^-1 or Hertz (Hz). 
+
+```
+f = 1/T
+
+```
+
+ 
+### 1.1.3 Time period (T)
+
+The time required to produced a full cicle its called time period of the wave (T). The IS of the Time period is the second (s). The frequency and the time period are reciprocal between them. 
+
+```
+T = 1/f
+```
+
+### 1.1.4 Wavelenght (λ)
+
+The minimun distance in which a sound wave repeats its called wavelenght. In a sound wave, the distance between the center of two consecutive compressions or two rarefactions are also denominated as wavelenght (λ, Lambda). The IS unit is the meter (m).
```

Comment:
```suggestion
 Wavelength is the distance from one crest to another, or from one trough to another, of a wave (which may be an electromagnetic wave, a sound wave, or any other wave). Crest is the highest point of the wave whereas the trough is the lowest. Since wavelength is distance/length, it is measured in units of lengths such as metres, centimetres, milimetres, nanometres, etc.
 
 Wavelength of light varies with colours, i.e., it is different for every colour. Red colour has the longest wavelength while violet has the least. The wavelength of UV radiation is shorter than violet light. Similarly, the wavelength of infra red radiation is longer than the wavelength of red light.
 
 Wavelength is inversely proportional to frequency. This means the longer the wavelength, lower the frequency. In the same manner, shorter the wavelength, higher will be the frequency. 
```

Reply:
outdated